cdc/sink/mysql/causality.go

```diff
@@ -53,21 +60,34 @@ func (c *causality) reset() {
 // detectConflict detects whether there is a conflict
 func (c *causality) detectConflict(keys [][]byte) (bool, int) {
```

Comment:
please add some comments about how to use the return value ( bool , int ).

Reply:
Done